ydb/core/mind/hive/hive_impl.cpp

```diff
@@ -1780,7 +1780,7 @@ bool THive::IsTabletMoveExpedient(const TTabletInfo& tablet, const TNodeInfo& no
     return result;
 }
 
-void THive::FillTabletInfo(NKikimrHive::TEvResponseHiveInfo& response, ui64 tabletId, const TLeaderTabletInfo *info, const NKikimrHive::TEvRequestHiveInfo &req) {
+void THive::FillTabletInfo(NKikimrHive::TEvResponseHiveInfo& response, ui64 tabletId, const TLeaderTabletInfo *info, const NKikimrHive::TEvRequestHiveInfo &req, TInstant restartsBarrierTime) {
```

Comment:
is it possible to put the barrier argument deeper down? TActivationContext::Now() is relatively a cheap call.

Reply:
it doesn't have to be static forever